scripts/TEST.vcf

```diff
@@ -0,0 +1,89 @@
+##fileformat=VCFv4.2
+##FILTER=<ID=PASS,Description="All filters passed">
+##INFO=<ID=MQ,Number=1,Type=Integer,Description="RMS mapping quality (normal sample)">
+##INFO=<ID=MQ0,Number=1,Type=Integer,Description="Number of MAPQ=0 reads (normal sample)">
+##INFO=<ID=CAL,Number=.,Type=String,Description="List of callers making this call">
+##INFO=<ID=Hotspot,Number=0,Type=Flag,Description="In mutational hotspot, as defined by Memorial Sloan Kettering Cancer Center based on Chang et al. 2017; see https://www.cancerhotspots.org">
```

Comment:
Hmm, should we make the description editable? We certainly will be using this in prod, but for research project, hot spots could change

Reply:
That sounds good! I'm not sure what you have in mind to make it editable, though, could you elaborate?

Reply:
Um, I suppose maybe give it a generic default description, like "Important variant marked as a hot spot; is an exception to consensus rule", but if a person gives their own string, something like you have now, then replace the description with that. In short, I propose making hot spot description an optional arg

Reply:
Ok!